ext/openssl/ossl_ssl.c

```diff
@@ -1046,6 +1046,34 @@ ossl_sslctx_set_ciphers(VALUE self, VALUE v)
     return v;
 }
 
+/*
+ * call-seq:
+ *    ctx.enable_fallback_scsv() => nil
+ *
+ * Activate TLS_FALLBACK_SCSV for this context.
+ * See RFC 7507.
+ */
+static VALUE
+ossl_sslctx_enable_fallback_scsv(VALUE self)
+{
+#ifdef SSL_MODE_SEND_FALLBACK_SCSV
+    SSL_CTX *ctx;
+    long modes;
+
+    GetSSLCTX(self, ctx);
+    if(!ctx){
+        rb_warning("SSL_CTX is not initialized.");
+        return Qnil;
+    }
+
+    modes = SSL_CTX_get_mode(ctx);
+    modes |= SSL_MODE_SEND_FALLBACK_SCSV;
+    SSL_CTX_set_mode(ctx, modes);
```

Comment:
SSL_CTX_set_mode() preserves the previously set flags, so just `SSL_CTX_set_mode(ctx, SSL_MODE_SEND_FALLBACK_SCSV)` is fine.

Reply:
Fixed